datascience-notebook/Dockerfile

```diff
@@ -5,10 +5,6 @@ FROM $BASE_CONTAINER
 
 LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
-# Set when building on Travis so that certain long-running build steps can
-# be skipped to shorten build time.
-ARG TEST_ONLY_BUILD
```

Comment:
This is not set by GitHub and we haven't seen any issues with `datascience-notebook` build so far.
So I think the best choice would be to remove this code.